lib/rosette/integrations/smartling-integration/smartling_puller.rb

```diff
@@ -1,135 +1,217 @@
 # encoding: UTF-8
 
+require 'set'
+require 'concurrent'
+
 module Rosette
   module Integrations
     class SmartlingIntegration < Integration
       class SmartlingPuller
 
-        attr_reader :rosette_config, :smartling_apis, :completed_files_map, :locale
+        DEFAULT_THREAD_POOL_SIZE = 15
```

Comment:
this thread pool size is different than above. coincidence? on purpose because they're different operations?